regulation-worker/cmd/main.go

```diff
@@ -34,12 +35,16 @@ func main() {
 }
 
 func Run(ctx context.Context) {
-
-	router := delete.NewRouter(&kvstore.KVDeleteManager{}, &batch.BatchManager{}, &api.APIManager{})
+	transformerURL := config.GetEnv("DEST_TRANSFORM_URL", "http://localhost:9090")
+	apiManager := api.APIManager{
+		Client:           &http.Client{},
+		DestTransformURL: transformerURL,
+	}
+	router := delete.NewRouter(&kvstore.KVDeleteManager{}, &batch.BatchManager{}, &apiManager)
 	svc := service.JobSvc{
 		API: &client.JobAPI{
 			WorkspaceID: config.GetEnv("workspaceID", "1001"),
-			URLPrefix:   config.GetEnv("urlPrefix", "https://api.rudderlabs.com:35359"),
+			URLPrefix:   config.GetEnv("urlPrefix", "http://localhost:35359"),
```

Comment:
Does `http://localhost:35359` make sense as default?

Reply:
Changing it.